src/api/app/views/webui2/webui/package/_delete_file_dialog.html.haml

```diff
@@ -0,0 +1,11 @@
+.modal.fade{ id: "delete-file-modal-#{file_id}", tabindex: -1, role: 'dialog', aria: { labelledby: 'delete-modal-label', hidden: true } }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %h5.modal-title
+          Really remove file?
```

Comment:
Again, consistency: `Delete file?`

Reply:
And the action button should preferably also have 'Delete' on it.

Reply:
Again, is consistent with the text we had before...